MastodonSDK/Sources/MastodonCore/Service/API/APIService+Mute.swift

```diff
@@ -21,6 +21,36 @@ extension APIService {
         let isMuting: Bool
     }
     
+    @discardableResult
+    public func getMutes(
+        authenticationBox: MastodonAuthenticationBox
+    ) async throws -> Mastodon.Response.Content<[Mastodon.Entity.Account]> {
+        let managedObjectContext = backgroundManagedObjectContext
+        
+        let response = try await Mastodon.API.Account.mutes(
+            session: session,
+            domain: authenticationBox.domain,
+            authorization: authenticationBox.userAuthorization
+        ).singleOutput()
+        
+        let userIDs = response.value.map { $0.id }
+        let predicate = NSPredicate(format: "%K IN %@", #keyPath(MastodonUser.id), userIDs)
```

Comment:
Please use the `MastodonUser.predicate(domain:ids:)` as predicate here. That will guarantee all users locate in specific domain. Users in different domain may use the same ID.

Reply:
I'Ve update this.